cdap-ui/yarn.lock

```diff
@@ -5402,6 +5417,44 @@ cypress@3.3.2:
     url "0.11.0"
     yauzl "2.10.0"
 
+cypress@^3.4.1:
```

Comment:
did you run yarn install? The version here seemed to be different than the one in package.json?

Reply:
This seem to be coming from cypress-visual-regression npm package added.